Formula/s/simutrans.rb

```diff
@@ -31,20 +31,21 @@ class Simutrans < Formula
   depends_on "freetype"
   depends_on "libpng"
   depends_on "sdl2"
+  depends_on "zstd"
 
   uses_from_macos "curl"
   uses_from_macos "unzip"
 
   fails_with gcc: "5"
 
   resource "pak64" do
-    url "https://downloads.sourceforge.net/project/simutrans/pak64/123-0/simupak64-123-0.zip"
-    sha256 "b8a0a37c682d8f62a3b715c24c49bc738f91d6e1e4600a180bb4d2e9f85b86c1"
+    url "https://downloads.sourceforge.net/project/simutrans/pak64/124-0/simupak64-124-0.zip"
+    sha256 "0defc5e7ce4c2c3620b621d94d0735dacc3ff13b1af24dee3a127ca76603b2a3"
   end
 
   def install
     # These translations are dynamically generated.
-    system "./get_lang_files.sh"
+    system "./tools/get_lang_files.sh"
 
     system "cmake", "-B", "build", "-S", ".", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-B", "build", "-S", ".", *std_cmake_args, "-DSIMUTRANS_USE_REVISION=#{version}"
```
Not sure of a way to programmatically get the revision from the URL, but this'll get around it. However, now it fails after building with "No such file or directory - build/makeobj/makeobj"

Reply:
The final build steps now need to be:
```
    bin.install "build/src/makeobj/makeobj"
    bin.install "build/src/nettools/nettool"
```

Reply:
> Not sure of a way to programmatically get the revision from the URL, but this'll get around it.

I was thinking about it, but it should be revision rather than version, right?